addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayDeviceHandler.java

```diff
@@ -466,14 +467,15 @@ private void zwayUnsubscribeOpenHabItem(Item openHABItem) {
 
     @Override
     public void handleUpdate(ChannelUID channelUID, State newState) {
+        // Deprecated, therefore deactivated
         // Refresh update time
-        logger.debug("Handle update for channel: {} with new state: {}", channelUID.getId(), newState.toString());
-
-        refreshLastUpdate();
+        // logger.debug("Handle update for channel: {} with new state: {}", channelUID.getId(), newState.toString());
+        // refreshLastUpdate();
```

Comment:
Do we really want to have commented out Code in here?

Reply:
Agree. Code removed